pkg/exchange/binance/parse.go

```diff
@@ -558,36 +605,214 @@ type ContinuousKLineEvent struct {
+// 			"bc":"50.12345678"            // Balance Change except PnL and Commission
+// 		  },
+// 		  {
+// 			"a":"BUSD",           
+// 			"wb":"1.00000000",
+// 			"cw":"0.00000000",         
+// 			"bc":"-49.12345678"
+// 		  }
+// 		],
+// 		"P":[
+// 		  {
+// 			"s":"BTCUSDT",            // Symbol
+// 			"pa":"0",                 // Position Amount
+// 			"ep":"0.00000",            // Entry Price
+// 			"cr":"200",               // (Pre-fee) Accumulated Realized
+// 			"up":"0",                     // Unrealized PnL
+// 			"mt":"isolated",              // Margin Type
+// 			"iw":"0.00000000",            // Isolated Wallet (if isolated position)
+// 			"ps":"BOTH"                   // Position Side
+// 		  }，
+// 		  {
+// 			  "s":"BTCUSDT",
+// 			  "pa":"20",
+// 			  "ep":"6563.66500",
+// 			  "cr":"0",
+// 			  "up":"2850.21200",
+// 			  "mt":"isolated",
+// 			  "iw":"13200.70726908",
+// 			  "ps":"LONG"
+// 		   },
+// 		  {
+// 			  "s":"BTCUSDT",
+// 			  "pa":"-10",
+// 			  "ep":"6563.86000",
+// 			  "cr":"-45.04000000",
+// 			  "up":"-1423.15600",
+// 			  "mt":"isolated",
+// 			  "iw":"6570.42511771",
+// 			  "ps":"SHORT"
+// 		  }
+// 		]
+// 	  }
+//   }
+
+type AccountConfig struct {
+	Symbol  string  `json:"s"`
+	Leverage  fixedpoint.Value  `json:"l"`
+}
+
+type AccountConfigUpdateEvent struct {
+	EventBase
+	Transaction  int64  `json:"T"`
+
+	AccountConfig  AccountConfig `json:"ac"`
+}
+
+// {
+//     "e":"ACCOUNT_CONFIG_UPDATE",       // Event Type
+//     "E":1611646737479,                 // Event Time
+//     "T":1611646737476,                 // Transaction Time
+//     "ac":{                              
+//     "s":"BTCUSDT",                     // symbol
+//     "l":25                             // leverage
+//     }
+// }  
+
+
+type OrderTrade struct {
+	Symbol  			string  `json:"s"`
+	ClientOrderID		string `json:"c"`
+	Side   				string `json:"S"`
+	OrderType         	string `json:"o"`
+	TimeInForce     	string `json:"f"`
+	OriginalQuantity 	string `json:"q"`
+	OriginalPrice      	string `json:"p"`
+
+	AveragePrice 		string `json:"ap"`
+	StopPrice  			string `json:"sp"`
+	CurrentExecutionType string `json:"x"`   
+	CurrentOrderStatus 	 string `json:"X"`     
+	
+	OrderId 						int64 `json:"i"`
+	OrderLastFilledQuantity			string `json:"l"`
+	OrderFilledAccumulatedQuantity	string `json:"z"`
+	LastFilledPrice 			  	string `json:"L"`
+
+	CommissionAmount string `json:"n"`
+	CommissionAsset  string `json:"N"`
+
+	OrderTradeTime	int64  `json:"T"`
+	TradeId  		int64 `json:"t"`
+
+	BidsNotional  	string `json:"b"`
+	AskNotional  	string `json:"a"`
+
+	IsMaker 		bool `json:"m"`
+	IsReduceOnly 	bool` json:"r"`
+
+	StopPriceWorkingType 	string `json:"wt"`
+	OriginalOrderType 		string `json:"ot"`
+	PositionSide 	string `json:"ps"`
+	RealizedProfit 	string `json:"rp"`
+
+}
+
+type OrderTradeUpdateEvent struct {
+	EventBase
+	Transaction int64 	`json:"T"`
+	OrderTrade 		OrderTrade	`json:"o"`
```

Comment:
this means it contains Order data or Trade data?

Reply:
Yes, it's similar to the ExecutionReportEvent. But with totally different json string fields.

e.g.,
 - Stop price.

So that, we can not merge them.

Reply:
thanks, can you add that in the comment?

Reply:
Certainly, done!